rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
I'd like to defer this also. I didn't make the decision to put this in it's own file in the first place and I don't want to argue the thinking behind that decision.